{"total_count": 1, "incomplete_results": false, "items": [{"url": "https://api.github.com/repos/smeso/MTPwn/issues/3", "repository_url": "https://api.github.com/repos/smeso/MTPwn", "labels_url": "https://api.github.com/repos/smeso/MTPwn/issues/3/labels{/name}", "comments_url": "https://api.github.com/repos/smeso/MTPwn/issues/3/comments", "events_url": "https://api.github.com/repos/smeso/MTPwn/issues/3/events", "html_url": "https://github.com/smeso/MTPwn/issues/3", "id": 391646421, "node_id": "MDU6SXNzdWUzOTE2NDY0MjE=", "number": 3, "title": "illegal instruction error\u00b2", "user": {"login": "K00p4", "id": 45657539, "node_id": "MDQ6VXNlcjQ1NjU3NTM5", "avatar_url": "https://avatars.githubusercontent.com/u/45657539?v=4", "gravatar_id": "", "url": "https://api.github.com/users/K00p4", "html_url": "https://github.com/K00p4", "followers_url": "https://api.github.com/users/K00p4/followers", "following_url": "https://api.github.com/users/K00p4/following{/other_user}", "gists_url": "https://api.github.com/users/K00p4/gists{/gist_id}", "starred_url": "https://api.github.com/users/K00p4/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/K00p4/subscriptions", "organizations_url": "https://api.github.com/users/K00p4/orgs", "repos_url": "https://api.github.com/users/K00p4/repos", "events_url": "https://api.github.com/users/K00p4/events{/privacy}", "received_events_url": "https://api.github.com/users/K00p4/received_events", "type": "User", "user_view_type": "public", "site_admin": false}, "labels": [], "state": "open", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 0, "created_at": "2018-12-17T10:26:44Z", "updated_at": "2018-12-17T10:27:49Z", "closed_at": null, "author_association": "NONE", "sub_issues_summary": {"total": 0, "completed": 0, "percent_completed": 0}, "active_lock_reason": null, "body": "like the issue on June, 11 this year I have the same problem:\r\n\r\nafter a short (incomplete) List of Folders I get the \"illegal instruction\" error.\r\n\r\ni changed the flags in the makefile and ran the following code: \r\n\r\n> run\r\n> list\r\n> x/3i $rip-3\r\n> print $rip\r\n> x/4i $rip-4\r\n> print $rip\r\n> x/5i  $rip-5\r\n\r\nWhen MTPwn get access via \r\n\r\n> gdb --batch --command=THEFILEYOUJUSTCREATED ./mtpwn (Filename = \"instructions\")\r\n\r\nI get the following message:\r\n\r\n**Thread 1 \"mtpwn\" received a signal SIGILL, Illegal instruction.\r\n0x0000555555555707 in ?? ( )\r\n1   ../sysdeps/x86_64/crti.S: File or Folder not found.\r\n0x555555555704  (Showing different digits maybe the adressing of the file???) \r\n0x555555555714  (Showing different digits maybe the adressing of the file???)\r\n0x555555555724  (Showing different digits maybe the adressing of the file???)\r\n0x555555555734  (Showing different digits maybe the adressing of the file???)\r\n0x555555555744  (Showing different digits maybe the adressing of the file???)\r\n0x555555555754  (Showing different digits maybe the adressing of the file???)\r\n0x555555555764  (Showing different digits maybe the adressing of the file???)\r\n0x555555555774  (Showing different digits maybe the adressing of the file???)\r\ninstructions:4: Error in sourced command file:\r\nNo symbol \"rip\" in current context.**\r\n\r\nSystem is Parrot OS 4.18\r\nlibmtp-dev is 1.1.13-1.1\r\n\r\nPhone is Samsung Galaxy A5 (2017) \r\n\r\nCan u help? :)", "reactions": {"url": "https://api.github.com/repos/smeso/MTPwn/issues/3/reactions", "total_count": 0, "+1": 0, "-1": 0, "laugh": 0, "hooray": 0, "confused": 0, "heart": 0, "rocket": 0, "eyes": 0}, "timeline_url": "https://api.github.com/repos/smeso/MTPwn/issues/3/timeline", "performed_via_github_app": null, "state_reason": null, "score": 1.0}]}